docs/basics/working-with-images.md

```diff
@@ -1,5 +1,3 @@
-import ImageDemo from './demos/image.demo.tsx'
-
 # Working with Images
 
 <!-- TODO add analysis section once it is merged -->
```

Comment:
@stropitek can i remove the TODO comment? I think the analysis part is covered in "Working with ROI".

Reply:
yes